lib/rubycritic/cli/options.rb

```diff
@@ -19,6 +19,16 @@ def parse
             @root = path
           end
 
+          opts.on('-b', '--branch BASE_BRANCH,FEATURE_BRANCH', 'Set branches to compare') do |branches|
+            self.base_branch, self.feature_branch = branches.split(',').map { |branch| String(branch) }
```

Comment:
Also, I think we can use only one branch name, and assume the currently checked-out branch is the "feature branch".
We can also assume a default "base branch", like master, since I guess it's the common case, and it'll make using this feature as simple as: `rubycritic -b`. Thoughts?